python/py-pymilter/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pymilter
+version             1.0.4
+revision            0
+distname            ${python.rootname}-${version}
+categories-append   mail
+license             Apache-2
+maintainers         {@BjarneDMat mathiesen.info:macintosh} openmaintainer
+description         Python interface to sendmail milter API
+long_description \
+    This is a python extension module to enable python scripts to \
+    attach to sendmail\'s libmilter functionality.  Additional python \
+    modules provide for navigating and modifying MIME parts, and \
+    sending DSNs or doing CBVs.
+
+platforms           darwin
+supported_archs     noarch
```

Comment:
this doesn't seem to be correct... at least from the [log](https://paste.macports.org/da706c221088) it appears that the extensions gets compiled, so there will be architecture-dependent files.